com.rioterm.Rio.yml

```diff
@@ -0,0 +1,54 @@
+---
+app-id: com.rioterm.Rio
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --device=dri
+  - --persist=xdg-config/rio
```

Comment:
this isn't valid.

(it will not do what you expect it to do)

Reply:
Removed, config now persists in `.var/app/com.rioterm.Rio/config` and upstream respects `XDG_CONFIG_HOME`